bnd-maven-plugin-parent/bnd-maven-plugin/pom.xml

```diff
@@ -0,0 +1,73 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>biz.aQute.bnd</groupId>
+		<artifactId>bnd-maven-plugin-parent</artifactId>
+		<version>2.5.0</version>
```

Comment:
The current version on Maven Central is 2.4.0. That was the experimental plugin that we abandoned.

This is a complete rewrite...